chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -1380,6 +1380,12 @@ pdfReader.searchAnnotations = Search Annotations
 pdfReader.noAnnotations = Create an annotation to see it in the sidebar
 pdfReader.noExtractedText = No extracted text
 pdfReader.addComment = Add comment
+pdfReader.annotationComment = Annotation comment
+pdfReader.annotationText = Annotation text
+pdfReader.manageTags = Click to manage tags
```

Comment:
It shouldn't be "click", presumably, if we're writing this for screen readers. "Manage tags"? "Manage tags for this annotation"?